src/docs/development/data-and-backend/state-mgmt/options.md

```diff
@@ -76,6 +76,13 @@ A state container approach familiar to many web developers.
   by Marcelo Glasberg
 * [Flutter meets Redux: The Redux way of managing Flutter applications state][],
   by Amir Ghezelbash
+  
+## Fish-Redux
+
+Fish Redux is an assembled flutter application framework based on Redux state management. It is suitable for building medium and large applications.
+
+* [Fish-Redux-Library][], by Alibaba
```

Comment:
Rather than linking to GitHub, what about linking to the package on pub.dev? Or include both links, but the package link first.

```
* [`fish-redux`][] package, by Alibaba
```